ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -247,7 +270,8 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: TableViewCell.identifier, for: indexPath) as! TableViewCell
-        cell.setup(title: "Cell #\(1 + indexPath.row)", accessoryType: .disclosureIndicator)
+        let imageView = UIImageView(image: UIImage(named: "excelIcon"))
```

Comment:
nit: Does this icon make sense here? At least in the tableviewcell demo, the cell title is a spreadsheet name

Reply:
it might not? the reason why i wanted some sort of color cell image is to show translucency of the tabbar background in dark mode. i do wonder if we should change this particular large title navigation controller demo to the next level and make it look more like office as we add more controls